src/entity/Source.ts

```diff
@@ -29,13 +29,15 @@ export interface SourceFlagsPublic {
   totalViews: number;
   totalPosts: number;
   totalUpvotes: number;
+  totalMembers: number;
 }
 
 export const defaultPublicSourceFlags: SourceFlagsPublic = {
   featured: false,
   totalViews: 0,
   totalPosts: 0,
   totalUpvotes: 0,
+  totalMembers: 0,
```

Comment:
We are gonna have to backfill too?

Reply:
Yes. We should have a script to fix this.